integration-tests/pipelines/rhtap-cli-e2e.yaml

```diff
@@ -38,6 +38,21 @@ spec:
     - name: konflux-test-infra-secret
       description: The name of secret where testing infrastructures credentials are stored.
       type: string
+    - name: acs_config
+      type: string
+      description: "The ACS option for rhtap-cli installation. Valid values are 'new' and 'hosted'."
+    - name: tpa_config
+      type: string
+      description: "The TPA option for rhtap-cli installation. Valid values are 'new' and 'hosted'."
+    - name: registry_config
```

Comment:
Should we consider any default value? Same question for other params as well

Reply:
I would like to add a `default.pict` in repository to specify the default values. For example [PR#203](https://github.com/redhat-appstudio/rhtap-e2e/pull/203/files)  So the modification of install.sh won't break the expected result. If we need to  consider any default value in task or pipline  yaml files in the future we can set default values.